moped-database/migrations/1644473675208_set_sequence_pointers/down.sql

```diff
@@ -0,0 +1 @@
+-- intentionally left blank
```

Comment:
There is no rolling back a set sequence value. At or about the time this PR is to be merged, the values in the up migration are right or they are not right. Moving them to another value in a down migration is not logically viable. Also, this PR can go stale, any inserts into any tables will require it to be re-inspected and updated.

This is OK right now because we're resetting the database, and would take real consideration and a maintenance window if we had live users and data.